docs/Documentation Generation.md

```diff
@@ -0,0 +1,84 @@
+# Doc generation using sphinx
+
+This document will help you create the documentation for entire VerticaPy library automatically using Sphinx.
+
+## Before you start
+
+- You need to clone the entire VerticaPy repo.
+- Open verticapy/_config/config.py and change the theme to sphinx (this ensures that the created plots/graphs are transparent which is needed to accessibility)
+
+```
+register_option(
+    Option(
+        "theme",
+        "sphinx",
+        "",
+        in_validator(["light", "dark", "sphinx"]),
+    )
+)
+```
+- Open a bash terminal and cd into ``VerticaPy/docs``. Then install all requirements
+
+```
+pip install -r requirements
+```
+- Additionally install pandoc
+
+```
+apt install pandoc
+```
+
+- Edit the ``replace_sphinx_dir.py`` file to change the replacement directory to the PWD (present working directory) of docs folder
```

Comment:
folder.

Reply:
This is my opinion:  There are a number of cases where comments don't end with a period.   In general, short, single sentences don't need to have them.   If a comment has more than one sentence, it should end with a period.  Everything in between is a judgement call.